stampit.js

```diff
@@ -8,25 +8,30 @@
  **/
 'use strict';
 var forEach = require('mout/array/forEach');
-var mixIn = require('mout/object/mixIn');
-var merge = require('mout/object/merge');
 var map = require('mout/array/map');
 var forOwn = require('mout/object/forOwn');
-var mixInChain = require('./mixinchain.js');
+var deepClone = require('mout/lang/deepClone');
 var slice = [].slice;
 
-// Avoiding JSHist W003 violations.
-var create, extractFunctions, stampit, compose, isStamp, convertConstructor;
+function isFunction(val) {
+  return typeof val === 'function';
+}
 
-create = function (o) {
-  if (arguments.length > 1) {
-    throw new Error('Object.create implementation only accepts the first parameter.');
-  }
-  function F() {}
+var mixer = require('./mixer');
+var mixIn = mixer.getMixin(); // Regular mixin function.
+var mixInFunctions = mixer.getMixin(isFunction); // mixin for functions only.
+var mixInChainFunctions = mixer.getMixin(isFunction, true); // mixin for functions including prototype chain.
```

Comment:
This function signature is a [boolean trap](http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html). Possible solution: named options hash.